README.md

```diff
@@ -1 +1,33 @@
-# plugin_sublime
\ No newline at end of file
+![Pieces Sublime Plugin](https://camo.githubusercontent.com/69c990240f877927146712d45be2f690085b9e45b4420736aa373917f8e0b2c8/68747470733a2f2f73746f726167652e676f6f676c65617069732e636f6d2f7069656365735f7374617469635f7265736f75726365732f7066645f77696b692f5049454345535f4d41494e5f4c4f474f5f57494b492e706e67)
+
+# Pieces Sublime Plugin
```

Comment:
do we have the ability to right click, save an asset as well? and are there any other features in here that we are listing like search? that we might want to add as well?

Reply:
We have the ability to right click to create an asset